### PR TITLE
import_image will be as well exit plugin, and will run only if build …

### PR DESCRIPTION
**Pull request by rcerven**
…was successful

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: tests/plugins/test_import_image.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def prepare(insecure_registry=None, namespace=None, primary_images_tag_conf=DEFA
     tasker = DockerTasker()
     workflow = DockerBuildWorkflow(SOURCE, "test-image")
     setattr(workflow, 'builder', X())
+    flexmock(workflow, build_process_failed=build_process_failed)
+#    setattr(workflow, 'build_process_failed', build_process_failed)
```
Comment (twaugh) [inline tests/plugins/test_import_image.py]: Stale?

Comment (rcerven) [reply tests/plugins/test_import_image.py]: right will delete it

**Comment by rcerven** [comment]
deleted commented-out code

**Comment by lcarva** [comment]
Updated release notes draft:
> - `import_image` is now also an `ExitPlugin`

**Pull request merged**